kernelci/build.py

```diff
@@ -273,6 +276,12 @@ def generate_config_fragment(frag, kdir):
             f.write(kernel_config + '\n')
 
 
+slim_skip_frags = [
+    "tools/testing/selftests/lkdtm/config",
+    "tools/testing/selftests/cpufreq/config"
+]
+
+
```

Comment:
How about defining fragments in YAML which disable some configs and then have e.g. `kselftest+no-lkdtm+no-cpufreq`?  That way we don't need to pile up more logic on this kselftest corner-case fragment.  All the other fragments are entirely defined in YAML and I think it would be better if we also tried to remove the kselftest special case.

Note: A more longer-term solution I think would be to use the generic way of making the kselftest config with `make kselftest-merge` and some way of skipping fragments in the upstream Makefile.

Reply:
That would be nice but it's much more work than I'm likely to be able to do in the medium term - we don't have the infrastructure to negatively apply config fragments like this. Getting something done upstream is probably interesting too, it's not clear that it'd even be recommended to put all the config fragments in one kernel as standard (for the reasons we're seeing here).